### DISCUSSION TITLE
刚初始化的项目，打完包87M，这个大吗

### DISCUSSION THREAD

**Discussion by show110695** [alex8088/electron-vite#323] [Q&A]
![image](https://github.com/alex8088/electron-vite/assets/57932634/1d7938ed-d4c5-4732-ade8-bfdceb88e8ba)

"dependencies": {
    "@electron-toolkit/preload": "^2.0.0",
    "@electron-toolkit/utils": "^2.0.0"
  },
  "devDependencies": {
    "@electron-toolkit/eslint-config": "^1.0.1",
    "@electron-toolkit/eslint-config-ts": "^1.0.0",
    "@electron-toolkit/tsconfig": "^1.0.1",
    "@rushstack/eslint-patch": "^1.3.3",
    "@types/node": "^18.17.5",
    "@vitejs/plugin-vue": "^4.3.1",
    "@vue/eslint-config-prettier": "^8.0.0",
    "@vue/eslint-config-typescript": "^11.0.3",
    "electron": "^25.6.0",
    "electron-builder": "^24.6.3",
    "electron-vite": "^1.0.27",
    "eslint": "^8.47.0",
    "eslint-plugin-vue": "^9.17.0",
    "less": "^4.2.0",
    "prettier": "^3.0.2",
    "typescript": "^5.1.6",
    "vite": "^4.4.9",
    "vue": "^3.3.4",
    "vue-tsc": "^1.8.8"
  }

**Comment by alex8088** [comment]
macOS中这是electron最小的包体积了

**Comment by luckyLocas** [comment]
很正常了，现在随便一个手机APP都多大了
